app/models/user.rb

```diff
@@ -35,11 +35,15 @@ def login_token
   def name
     "#{first_name} #{last_name}"
   end
-
+  
   def email
     identity.try :email
   end
-
+  
+  def active_since
```

Comment:
ammendment: I didn't know this, but you cannot use `alias_method` with ActiveRecord attributes... However, Rails has [`alias_attribute`](http://api.rubyonrails.org/classes/Module.html#method-i-alias_attribute) which accomplishes the same :smile: